app/client/components/contactus/contactUsConfig.ts

```diff
@@ -0,0 +1,287 @@
+interface BaseTopic {
+  id: string;
+  name: string;
+  selfServiceBox?: SelfServiceBox;
+  editableSubjectLine?: boolean;
+}
+
+interface SelfServiceBox {
+  text: string;
+  linkText: string;
+  href: string;
+  noForm?: boolean;
+}
+
+export interface SubTopic extends BaseTopic {
+  subsubtopics?: BaseTopic[];
+  subsubTopicsTitle?: string;
+}
+
+export interface Topic extends BaseTopic {
+  enquiryLabel: string;
+  subtopics?: SubTopic[];
+  subTopicsTitle?: string;
+}
+
+export const contactUsConfig: Topic[] = [
+  {
+    id: "delivery",
+    name: "Delivery",
+    enquiryLabel: "your delivery problem",
+    subTopicsTitle: "sub topic title here",
+    subtopics: [
+      {
+        id: "s1",
+        name: "I'm going on holiday and need to pause deliveries"
+      },
+      {
+        id: "s2",
+        name: "I need to report a problem with my delivery"
+      },
+      {
+        id: "s3",
+        name: "I need to update my delivery details"
+      },
+      {
+        id: "s4",
+        name: "Something else",
+        editableSubjectLine: true
+      }
+    ]
+  },
+  {
+    id: "billing",
+    name: "Billing",
+    enquiryLabel: "your billing enquiry",
+    subTopicsTitle: "sub topic title here",
+    subtopics: [
+      {
+        id: "s5",
+        name: "I want to update my payment details",
+        selfServiceBox: {
+          text:
+            "Did you know you can suspend your deliveries online by logging in below and selecting ‘Manage Subscription’? It’s easy to use and means you don’t have to wait for a response.",
+          linkText: "Go to your account",
+          href: "/billing"
+        }
+      },
+      {
+        id: "s6",
+        name: "My payment has failed"
+      },
+      {
+        id: "s7",
+        name: "I would like to see my invoices and payments"
+      },
+      {
+        id: "s8",
+        name: "I would like to cancel my payment"
+      },
+      {
+        id: "s9",
+        name: "Something else",
+        editableSubjectLine: true
+      }
+    ]
+  },
+  {
+    id: "vouchers",
+    name: "newspaper vouchers",
```

Comment:
Other names seem to start with upper case.

Reply:
I'll update the copy for the config file here and in a subsequent pr that moves this config over to the server